cmd/percona-dbaas/psmdb/create.go

```diff
@@ -68,8 +69,23 @@ var createCmd = &cobra.Command{
 		if len(args) >= 2 {
 			rsName = args[1]
 		}
+		if len(*labels) > 0 {
+			match, err := regexp.MatchString("([a-zA-Z0-9_]+=[a-zA-Z0-9_]+)|([a-zA-Z0-9_]+=[a-zA-Z0-9_]+,)+([a-zA-Z0-9_]+=[a-zA-Z0-9_]+)", *labels)
```

Comment:
1) only properly formated key-values allowed
2) it can be more than two labels
```suggestion
			match, err := regexp.MatchString("^(([a-zA-Z0-9_]+=[a-zA-Z0-9_]+)(,|$))+$", *labels)
```

Reply:
done